charts/latest/csi-driver-smb/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
     tag: v2.8.0
     pullPolicy: IfNotPresent
   csiproxy:
-    repository: ghcr.io/kubernetes-sigs/sig-windows/csi-provisioner
+    repository: ghcr.io/kubernetes-sigs/sig-windows/csi-proxy
```

Comment:
what's this change?

Reply:
Fix: CSI proxy for windows defaults used wrong image

Reply:
`ghcr.io/kubernetes-sigs/sig-windows/csi-proxy:v1.1.2` does not exist

```
# docker pull ghcr.io/kubernetes-sigs/sig-windows/csi-proxy:v1.1.2
v1.1.2: Pulling from kubernetes-sigs/sig-windows/csi-proxy
no matching manifest for linux/amd64 in the manifest list entries
```

Reply:
ofcourse not, it is a windows image